libraries/chain/include/eosio/chain/kv_chainbase_objects.hpp

```diff
@@ -46,29 +44,19 @@ namespace eosio { namespace chain {
          kv_object,
          indexed_by<ordered_unique<tag<by_id>, member<kv_object, kv_object::id_type, &kv_object::id>>,
                     ordered_unique<tag<by_kv_key>,
-                                   composite_key<kv_object, member<kv_object, name, &kv_object::database_id>,
+                                   composite_key<kv_object,
                                                  member<kv_object, name, &kv_object::contract>,
                                                  member<kv_object, shared_blob, &kv_object::kv_key>>,
-                                   composite_key_compare<std::less<name>, std::less<name>, unsigned_blob_less>>>>;
-
-   inline void use_rocksdb_for_disk(chainbase::database& db) {
```

Comment:
This function made sure that KV data was stored either in chainbase or in RocksDB, but not both at the same time. I couldn't find a replacement for this functionality anywhere else in this PR.